scripts/observers/Makefile

```diff
@@ -1,11 +1,14 @@
 # Help section
 help:
 	@echo "Available targets:"
-	@echo "  config  - Configure the local testnet"
-	@echo "  start   - Start the local testnet"
-	@echo "  stop    - Stop the local testnet"
-	@echo "  clean   - Clean up the files of the local testnet"
-	@echo "  cluster - Start an Elasticsearch cluster on port 9200"
+	@echo "  config        - Configure the local testnet"
+	@echo "  start         - Start the local testnet"
+	@echo "  stop          - Stop the local testnet"
```

Comment:
suggestion to add in a next PR: add `install` command to handler prerequisites, like python install requirements, `screen`, ... , in order to be able to run the entire setup more easily if trying for the first time

Reply:
maybe also commands to handle the entire setup like:
`full-start: start cluster monitor-start`
`full-stop: stop cluster-stop monitor-stop`

Reply:
Will make a task for this

Reply:
[MX-14453]